services/federator/test/unit/Test/Federator/Monitor.hs

```diff
@@ -119,7 +119,7 @@ withSilentMonitor reloads settings = do
   tlsVar <- liftIO $ newIORef (error "TLSSettings not updated before being read")
   void . ContT $
     bracket
-      (runSem (mkMonitor runSemE tlsVar settings))
+      (runSem (mkMonitor runSemE (atomicWriteIORef tlsVar) settings))
```

Comment:
this is not a tlsVar any more, no?

Reply:
It is an `IORef` same as earlier, see 3 lines above.